### ISSUE TITLE
could u plz add the vpn options?

### ISSUE THREAD

**Issue by FolStone** [fribbels/Fribbels-Epic-7-Optimizer#183]
its hard to log in game direct, but I can not use optimizer by vpn

**Comment by login0false** [comment]
Yes, there should be some methods available to make the importer work with at least some kinds of VPNs (or a system proxy). I cannot log into the game without one most of the time where I live, effectively deleting the auto importer functionality.
